pdfium-binary.pri

```diff
@@ -1,5 +1,9 @@
 win32 {
-    LIBS += -L$$PWD/win32precompiled
+    !contains(QMAKE_TARGET.arch, x86_64) {
```

Comment:
Nit-pick: I think not using not is easier to read (humans are bad at negation), so I'd prefer the condition inverted. But it's really no issue.